frontend/src/components/search/SearchFilterAgency.tsx

```diff
@@ -0,0 +1,9 @@
+import { SearchFilterAccordion } from "src/components/search/SearchFilterAccordion/SearchFilterAccordion";
+import { agencyFilterList } from "./SearchFilterAccordion/filterJSONLists/agencyFilterList";
```

Comment:
Is there a reason to put the JSON data in a separate file? The other filters just have their data in the component. This file is so short, I don't think it'd be a big deal for it to also include the data. We should use a consistent approach, and it'd seem silly to separate the other filters into tiny component and data files.

Reply:
I thought we're planning to hardcode the entire agnecy filter list ? What we have now is only a small sample of the real thing (I pulled from the taxonomy sheet : https://docs.google.com/spreadsheets/d/1M14vX-kDakn0QHhYkUnIasBnTFUfU73BKXXxAfCxtzQ/edit#gid=1891194558)

The real thing will be like 1000 LOC - I'm not sure that makes sense to keep in the same file?